test/seeds/test17.ged

```diff
@@ -0,0 +1,257 @@
+0 HEAD
```

Comment:
Just make sure that this doesn't break other people's testing if they used a test17.ged as well